src/nodes.coffee

```diff
@@ -1305,14 +1305,24 @@ exports.Call = class Call extends Base
   # expands the range on the left, but not the right.
   updateLocationDataIfMissing: (locationData) ->
     if @locationData and @needsUpdatedStartLocation
-      @locationData.first_line = locationData.first_line
-      @locationData.first_column = locationData.first_column
-      @locationData.range[0] = locationData.range[0]
+      @locationData = Object.assign {},
```

Comment:
Ok updated to not mutate `@locationData`

I think by starting from eg `@locationData = {}` we would've lost `last_line`/`last_column`, so I used an `Object.assign()`-based pattern

Reply:
Don't we support `{...o}`?

Reply:
CoffeeScript does but Node 6 doesn't. The compiler is keeping compatibility with Node 6+.